core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -739,7 +739,8 @@ public FileInfo getFileInfo(AlluxioURI path, GetStatusOptions options)
     Metrics.GET_FILE_INFO_OPS.inc();
     LockingScheme lockingScheme =
         createLockingScheme(path, options.getCommonOptions(), InodeTree.LockMode.READ);
-    try (JournalContext journalContext = createJournalContext();
+    try (BlockDeletionContext blockDeleter = new DefaultBlockDeletionContext(mBlockMaster);
```

Comment:
NIT: consistency of the variable name? The class says "Context", but the variable does not.